cli/DefaultsConfigurator.hpp

```diff
@@ -22,6 +22,7 @@
 #include <unordered_map>
 #include <vector>
 
+#include "cli/CliConfig.h"
```

Comment:
I agree with @zuyu because ``QUICKSTEP_HAVE_LIBNUMA`` is a result of whether NUMA libraries are present on the system or not (Ref: https://github.com/pivotalsoftware/quickstep/blob/master/storage/CMakeLists.txt#L41). Either we repeat the same check in cli/CMakeLists.txt or just use whatever is already done in storage module.